fdbserver/BlobManager.actor.cpp

```diff
@@ -5145,6 +5134,22 @@ ACTOR Future<Void> bgConsistencyCheck(Reference<BlobManagerData> bmData) {
 	}
 }
 
+ACTOR Future<Void> backupManifest(Reference<BlobManagerData> bmData) {
+	if (g_network->isSimulated() && g_simulator->speedUpSimulation) {
```

Comment:
Is this so this stops doing things for quiet database?

Reply:
I simply port it from bgcc. so far I don't want to have it for simulation yet. will remove it when I start working on tests.